README.org

```diff
@@ -61,3 +61,24 @@
    - Matthew Fidler <matthew.fidler@gmail.com>
    - Michael Markert <markert.michael@gmail.com>
    - Jan Path <jan@jpath.de>
+
+** Project status
+⚠️ This project currently has no "official" maintainer.
+
+`evil-numbers` is licensed under the GPL so that anyone can modify and
+distribute the source code. As [[http://catb.org/~esr/writings/homesteading/homesteading/ar01s03.html][ESR notes]], "Nothing prevents half a
+dozen different people from taking any given open-source product,
+duplicating the sources, running off with them in different
+evolutionary directions..."  The originator appears to have abandoned
```

Comment:
I'm still not convinced I want to explain the concept of forks in this README. Seems kinda off-topic. I'm fine mentioning that cofi seems to have abondend this and that I am currently maintaining this fork and am happy to accept PRs, but not sure about this.